__tests__/demo_service/invoke-service.sh

```diff
@@ -1 +1,6 @@
-docker run --rm -v $(dirname $(realpath $0)):/var/task lambci/lambda:build-ruby2.5 ruby -I. -rhandler -e "puts hello(event:{},context:{})"
+  docker run --rm                                                     \
+             --volume $(pwd):/var/task                                \
```

Comment:
The intent of `$(dirname $(realpath $0))` over `$(pwd)` was to make sure the directory where the script is defined is mounted as a volume -- regardless of the current working directory. Ie, if you invoke this script from the repo root, `$(pwd)` will be the repo root.